website/content/docs/upgrading/upgrade-to-1.8.0.mdx

```diff
@@ -0,0 +1,37 @@
+---
+layout: docs
+page_title: Upgrading to Vault 1.8.0 - Guides
+description: |-
+  This page contains the list of deprecations and important or breaking changes
+  for Vault 1.8.0. Please read it carefully.
+---
+
+# Overview
+
+This page contains the list of deprecations and important or breaking changes
+for Vault 1.8.0 compared to 1.7. Please read it carefully.
+
+## License Enhancements
+
+Licenses and EULA enhancements have been introduced in the Vault 1.8 release.
+These changes are important for Enterprise customers to review. They do not affect
+OSS users. Please see the [License](/docs/enterprise/license) documentation for more details.
+
+## Deprecations
+
+The following API endpoints have been deprecated and will be removed in a future release:
+
+* `PUT sys/license` to store licenses in storage; it is recommended to use
```

Comment:
I'd assumed we were going to deprecate /sys/license and /sys/license/signed altogether, not just PUT.  Rationale: 
- GET /sys/license/signed doesn't make sense with autoloading, the operator already has the signed license (we also didn't implement anything to make it work with autoloading)
- DELETE /sys/license serves no purpose when you don't have a stored license any longer, though naturally we may want to keep this one around longer than the others
- GET /sys/license isn't needed because GET /sys/license/status provides the same information and more, like what kind of license is in use; we could keep it around anyway, but I figured it was easier for people to reason about if we could just say that all /sys/license endpoints were deprecated

Reply:
I've made the description more generic. We can update it again as needed if we do need to be more precise as to method. cc @aartiiyengar